File: Ejercicio10-Condicionales.cpp
/*10. Mostrar los meses del año, pidiéndole al usuario un número entre (1-12), y mostrar el mes al que corresponde. */

#include<iostream>
using namespace std;
int main(){
    int mes;
    cout<<"Digite un numero [1-12]: ";
    cin>>mes;

    switch(mes){
        case 1: cout<<"\nEl mes es enero"; break;
        case 2: cout<<"\nEl mes es febrero"; break;
        case 3: cout<<"\nEl mes es marzo"; break;
        case 4: cout<<"\nEl mes es abril"; break;
        case 5: cout<<"\nEl mes es mayo"; break;
        case 6: cout<<"\nEl mes es junio"; break;
        case 7: cout<<"\nEl mes es julio"; break;
        case 8: cout<<"\nEl mes es agosto"; break;
        case 9: cout<<"\nEl mes es septiembre"; break;
        case 10: cout<<"\nEl mes es octubre"; break;
        case 11: cout<<"\nEl mes es noviembre"; break;
        case 12: cout<<"\nEl mes es diciembre"; break;
    }
    return 0;
}

File: Comparativa Python/Ejercicio10-Condicionales.py
"""10. Mostrar los meses del año, pidiéndole al usuario un número entre (1-12),
y mostrar el mes al que corresponde."""

mes = int(input('Digite un número entre [1-12]: '))

if mes == 1:
    print('El mes es enero')
elif mes == 2:
    print('El mes es febrero')
elif mes == 3:
    print('El mes es marzo')
elif mes == 4:
    print('El mes es abril')
elif mes == 5:
    print('El mes es mayo')
elif mes == 6:
    print('El mes es junio')
elif mes == 7:
    print('El mes es julio')
elif mes == 8:
    print('El mes es agosto')
elif mes == 9:
    print('El mes es septiembre')
elif mes == 10:
    print('El mes es octubre')
elif mes == 11:
    print('El mes es noviembre')
elif mes == 12:
    print('El mes es diciembre')
else:
    print('Valor digitado fuera de rango')
